plugins/callback/scalyr.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Martin Migasiewicz <migasiew.nk@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+callback: scalyr
+type: aggregate
+short_description: Sends task result events to Scalyr
+author: Martin Migasiewicz (@martinm82)
+version_added: 1.3.0
+description:
+  - This callback plugin will send task results as JSON formatted events to Scalyr.
+requirements:
+  - Whitelisting this callback plugin
+  - 'Create a Log Access Write Key in Scalyr (https://app.eu.scalyr.com/keys)'
+  - 'Define the Scalyr API URL and API key in ansible.cfg'
+  - 'Based on the splunk and sumologic callbacks'
+options:
+  url:
+    description: URL to the Scalyr API endpoint
+    default: https://app.eu.scalyr.com/api/addEvents
+    env:
+      - name: SCALYR_URL
+    ini:
+      - section: callback_scalyr
+        key: url
+    type: string
+  authtoken:
+    description: Token to authenticate the connection to Scalyr.
+    env:
+      - name: SCALYR_AUTHTOKEN
+    ini:
+      - section: callback_scalyr
+        key: authtoken
+    type: string
+'''
+
+EXAMPLES = '''
+examples: >
```

Comment:
```suggestion
examples: |
```
Or do you really want to remove newlines?

Reply:
Copy&paste. I haven't thought about what the difference is.